packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -170,83 +193,107 @@ export interface IBasePickerSuggestionsProps {
    * Function that specifies what to render when no results are found.
    */
   onRenderNoResultFound?: IRenderFunction<void>;
+
   /**
    * The text that should appear at the top of the suggestion box.
    */
   suggestionsHeaderText?: string;
+
   /**
    * The text that should appear at the top of the most recently used box.
    */
   mostRecentlyUsedHeaderText?: string;
+
   /**
-   * the text that should appear when no results are returned.
+   * The text that should appear when no results are returned.
    */
   noResultsFoundText?: string;
+
   /**
    * Suggestions root className.
    */
   className?: string;
+
   /**
    * Suggestions List className.
    */
   suggestionsClassName?: string;
+
   /**
    * ClassName for suggestion items.
    */
   suggestionsItemClassName?: string;
+
   /**
    * The text that should appear on the button to search for more.
    */
   searchForMoreText?: string;
+
   /**
    * The text that appears indicating to the use to force resolve the input
    */
   forceResolveText?: string;
+
   /**
    * The text to display while the results are loading.
    */
   loadingText?: string;
+
   /**
-   * The text to display while searching for more results in a limited sugesstions list.
+   * The text to display while searching for more results in a limited suggestions list.
    */
   searchingText?: string;
+
   /**
    * A renderer that adds an element at the end of the suggestions list it has more items than resultsMaximumNumber.
```

Comment:
Should it be `...suggestions list if it has...` instead of `...suggestions list it has...`?

Reply:
fixed